src/master-playlist-controller.js

```diff
@@ -238,8 +238,10 @@ export default class MasterPlaylistController extends videojs.EventTarget {
       if (master.mediaGroups && master.mediaGroups.AUDIO) {
         for (let groupKey in master.mediaGroups.AUDIO) {
           for (let labelKey in master.mediaGroups.AUDIO[groupKey]) {
+            /* eslint-disable no-warning-comments */
```

Comment:
We should probably keep the comment warnings. They don't break tests and it is good to be reminded that TODOs exist.